openedx/features/edly/utils.py

```diff
@@ -25,3 +32,22 @@ def decode_edly_user_info_cookie(encoded_cookie_data):
         dict
     """
     return jwt.decode(encoded_cookie_data, settings.EDLY_COOKIE_SECRET_KEY, algorithms=[settings.EDLY_JWT_ALGORITHM])
+
+def get_enabled_organizations(request):
+    """
+    Helper method to get linked organizations for request site.
+
+    Returns:
+        list: List of linked organizations for request site
+    """
+
+    if not waffle.switch_is_active(settings.ENABLE_EDLY_ORGANIZATIONS_SWITCH):
+        return get_organizations()
+    else:
+        try:
+          studio_site_edx_organization = [model_to_dict(request.site.studio_site.edx_organization)]
+        except EdlySubOrganization.DoesNotExist:
+          LOGGER.exception('No EdlySubOrganization found for site %s', request.site)
```

Comment:
Need to add test with for exception message `No EdlySubOrganization found for site ` using log capture
```python
with LogCapture(LOGGER_NAME, level=logging.INFO) as logger:
```

Reply:
Fancy! 🤝